applications/ipc_radio/README.rst

```diff
@@ -76,13 +76,16 @@ The Bluetooth Low Energy and IEEE 802.15.4 functionalities can operate simultane
 Sysbuild Kconfig options
 ========================
 
-To enable the firmware, use the sysbuild configuration of :kconfig:option:`SB_CONFIG_NRF_DEFAULT_IPC_RADIO`.
+To enable the firmware, use the sysbuild configuration ``SB_CONFIG_NETCORE_IPC_RADIO``.
 
 You can set the supported radio configurations using the following sysbuild Kconfig options:
 
-* :kconfig:option:`SB_CONFIG_NETCORE_IPC_RADIO_BT_HCI_IPC`
-* :kconfig:option:`SB_CONFIG_NETCORE_IPC_RADIO_BT_RPC`
-* :kconfig:option:`SB_CONFIG_NETCORE_IPC_RADIO_IEEE802154`
+* ``SB_CONFIG_NETCORE_IPC_RADIO_BT_HCI_IPC``
+* ``SB_CONFIG_NETCORE_IPC_RADIO_BT_RPC``
+* ``SB_CONFIG_NETCORE_IPC_RADIO_IEEE802154``
+
+.. note::
+   For |NCS| samples and applications, use the ``SB_CONFIG_NRF_DEFAULT_IPC_RADIO`` sysbuild configuration instead of ``SB_CONFIG_NETCORE_IPC_RADIO`` to enable to firmware.
```

Comment:
Note typo in ``to firmware``.
Does it make sense to mention what ``SB_CONFIG_NETCORE_IPC_RADIO`` is used for?
```suggestion
   For |NCS| samples and applications, use the ``SB_CONFIG_NRF_DEFAULT_IPC_RADIO`` sysbuild configuration to enable the firmware instead of ``SB_CONFIG_NETCORE_IPC_RADIO`` (which should only be used for production).
```

Reply:
The `SB_CONFIG_NRF_DEFAULT_IPC_RADIO` is present and used in all our (radio capable) samples, so I thought to mention the config and distingiush it from `SB_CONFIG_NETCORE_IPC_RADIO` to avoid confusion from the user perspective.